### DISCUSSION TITLE
İbax puan kullanımı ve uygılama

### DISCUSSION THREAD

**Discussion by mthnkrk** [IBAX-io/go-ibax#1771] [Soru Cevap - Türkçe]
İbax projesi genel olarak bakıldığında ağlar arasındaki transfer sürecinde zaman kaybınj önlemek amacıyla kurulma mantığıyla çalışıyor fakat gel görelim ki site alt yapı geliştiriceli ve veri tabanı uzmanlarının bu konuda yetersiz olacağını düşünüyorum.Bunun sebebi ise; proje ağ sistemi ve diğer konularda söyleninler içerisinde mantıklı ve yapılması istenilen yapılırsa güzel olacağa benziyor ama gel görelim ki birçok kullanıcıya vaad edilen retweet görevlerinde bile elde edilecek puanları sisteme yansıtmada bile başarısız kalınıyor ve hala çözüm bulunamıyırsa alt yapısal olarak sorun var demektir.

**Comment by Yusufknk** [comment]
Retweet görevleri için belirli sayıda API kontenjanı var bu kontenjan son retweet görevlerinde az sayıda kaldığı için ilk yapanlar doldurdu. Alınan hataların ve çoğu kişinin yapamamasinin nedeni bu
